### PR TITLE
Fix AR9X build and add it to regular commit tests.

### PR DESCRIPTION
**Pull request by schwabe**
Note that simu still crashes on startup in the readRtc function

### CODE REVIEW

File: radio/src/targets/simu/opentxsimulator.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ void OpenTxSimulator::start(QByteArray & ee, bool tests)
 
 void OpenTxSimulator::start(const char * filename, bool tests)
 {
-#if defined(PCBSKY9X) && !defined(REVX)
+#if defined(PCBSKY9X) && !defined(REVX) && !defined(AR9X)
```
Comment (bsongis) [inline radio/src/targets/simu/opentxsimulator.cpp]: perhaps && ROTARY_ENCODERS > 0 ? 
I am not sure

File: radio/src/targets/sky9x/i2c_driver.cpp [hunk 2]
```diff
@@ -18,6 +18,7 @@
  * GNU General Public License for more details.
  */
 
+#include "opentx.h"
 #include "i2c_driver.h"
```
Comment (bsongis) [inline radio/src/targets/sky9x/i2c_driver.cpp]: here some includes can be removed then

File: radio/src/targets/sky9x/rtc_ds3231_driver.cpp [hunk 3]
```diff
@@ -18,6 +18,7 @@
  * GNU General Public License for more details.
  */
 
+#include "opentx.h"
 #include "rtc.h"
```
Comment (bsongis) [inline radio/src/targets/sky9x/rtc_ds3231_driver.cpp]: the same here 
not sure though

**Comment by bsongis** [comment]
> Note that simu still crashes on startup in the readRtc function

Do you need help for this?

File: radio/src/targets/simu/opentxsimulator.cpp (hunk 1 above)
Comment (schwabe) [inline radio/src/targets/simu/opentxsimulator.cpp]: Neither am I

**Comment by schwabe** [comment]
No the second commit fixes the crashes in the simulator

File: radio/src/targets/simu/opentxsimulator.cpp (hunk 1 above)
Comment (bsongis) [inline radio/src/targets/simu/opentxsimulator.cpp]: I just checked, you should use NUM_ROTARY_ENCODERS > 0 and something is missing here:
#if defined(PCBSKY9X) && (defined(REVX) || defined(AR9X)
  #define MAX_ROTARY_ENCODERS          1
  #define NUM_ROTARY_ENCODERS          0
#elif defined(PCBSKY9X)
  #define MAX_ROTARY_ENCODERS          1
  #define NUM_ROTARY_ENCODERS          1
...

**Pull request merged**